src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -66,6 +67,11 @@ private static void setupDrivers(Application application) {
 		driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
 		DriverFeature.addDriver("Special line Simulator", driver);
 		DriverFeature.updateDriverInfo();
+
+		driver = new DriversComposite();
+		((DriversComposite)driver).addDriver(new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic"));
+		((DriversComposite)driver).addDriver(new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special"));
```

Comment:
I suggest to build composite from LineDriverAdapter with BasicLine and the LoggerDriver from Pull Request #5